src/iohumdrum.cpp

```diff
@@ -5479,21 +5517,11 @@ void HumdrumInput::addTrill(hum::HTp token)
     if (hasaccid) {
         accidval = stoi(accid);
         switch (accidval) {
-            case -1:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_f);
-                break;
-            case 0:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_n);
-                break;
-            case 1:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_s);
-                break;
-            case -2:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_ff);
-                break;
-            case 2:
-                trill->SetAccidupper(ACCIDENTAL_EXPLICIT_x);
-                break;
+            case -1: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_f); break;
+            case 0: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_n); break;
+            case 1: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_s); break;
+            case -2: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_ff); break;
+            case 2: trill->SetAccidupper(ACCIDENTAL_EXPLICIT_x); break;
```

Comment:
What do you get @lpugin when running clang-format on this file?  Not a big problem, but it will cause unnecessary version-control thrashing (which clang-format is supposed to minimize :-) 

```bash
$ clang-format --version
clang-format version 4.0.0 (tags/google/testing/2016-08-03)
```
(on MacOS Sierra)

And the .clang-format, which I presume is the same for everyone (as it comes from the `develop` branch).

```
---
Language:        Cpp
# BasedOnStyle:  WebKit
AccessModifierOffset: -4
AlignAfterOpenBracket: false
AlignEscapedNewlinesLeft: false
AlignOperands:   false
AlignTrailingComments: false
AllowAllParametersOfDeclarationOnNextLine: true
AllowShortBlocksOnASingleLine: false
AllowShortCaseLabelsOnASingleLine: true
# Verovio
AllowShortIfStatementsOnASingleLine: true
# Verovio
AllowShortLoopsOnASingleLine: true
# Verovio
AllowShortFunctionsOnASingleLine: Inline
AlwaysBreakAfterDefinitionReturnType: false
AlwaysBreakTemplateDeclarations: false
AlwaysBreakBeforeMultilineStrings: false
BreakBeforeBinaryOperators: All
BreakBeforeTernaryOperators: true
BreakConstructorInitializersBeforeComma: true
BinPackParameters: true
BinPackArguments: true
# Verovio
ColumnLimit:     120
# Verovio
ConstructorInitializerAllOnOneLineOrOnePerLine: true
ConstructorInitializerIndentWidth: 4
DerivePointerAlignment: false
ExperimentalAutoDetectBinPacking: false
IndentCaseLabels: true
IndentWrappedFunctionNames: false
IndentFunctionDeclarationAfterType: false
MaxEmptyLinesToKeep: 1
KeepEmptyLinesAtTheStartOfBlocks: true
NamespaceIndentation: Inner
ObjCBlockIndentWidth: 4
ObjCSpaceAfterProperty: true
ObjCSpaceBeforeProtocolList: true
PenaltyBreakBeforeFirstCallParameter: 19
PenaltyBreakComment: 300
PenaltyBreakString: 1000
PenaltyBreakFirstLessLess: 120
PenaltyExcessCharacter: 1000000
PenaltyReturnTypeOnItsOwnLine: 60
# Verovio
PointerAlignment: Right
SpacesBeforeTrailingComments: 1
Cpp11BracedListStyle: false
Standard:        Cpp03
IndentWidth:     4
TabWidth:        8
UseTab:          Never
BreakBeforeBraces: Stroustrup
SpacesInParentheses: false
SpacesInSquareBrackets: false
SpacesInAngles:  false
SpaceInEmptyParentheses: false
SpacesInCStyleCastParentheses: false
SpaceAfterCStyleCast: false
SpacesInContainerLiterals: true
SpaceBeforeAssignmentOperators: true
ContinuationIndentWidth: 4
CommentPragmas:  '^ IWYU pragma:'
ForEachMacros:   [ foreach, Q_FOREACH, BOOST_FOREACH ]
SpaceBeforeParens: ControlStatements
DisableFormat:   false
...
```